docs/Write-Plugin.md

```diff
@@ -0,0 +1,58 @@
+
+Have you got some question or are you seeking for a suggestion? We are more than happy to help you! Just open an issue in our [help](https://github.com/fastify/help) repository.
+
+Once you submit a plugin to our [ecosystem](https://github.com/fastify/fastify/blob/master/docs/Ecosystem.md) list, we will review your code and help you improve it if needed.
+
+## Documentation
+Documentation is extremely important, if your plugin is not well documented we will not accept it, and probably it is difficult that it will be used because of the lack of the documentation.<br>
+If you want to see some good examples on how document a plugin take a look at:
+- [`fastify-cookie`](https://github.com/fastify/fastify-cookie)
+- [`point-of-view`](https://github.com/fastify/point-of-view)
+- [`fastify-compress`](https://github.com/fastify/fastify-compress)
+- [`under-pressure`](https://github.com/fastify/under-pressure)
+- [`fastify-caching`](https://github.com/fastify/fastify-caching)
+
+## License
+You can license your plugin as you prefer, we do not enforce any kind of license.<br>
+Our recommendation is to license your code as [MIT](https://choosealicense.com/licenses/mit/), in this way no one will have legal concerns when using it.
+
+## Examples
+Always put an example file in your repository, they are very helpful for users and give a very fast way to test your plugin. Your users will be grateful.
+
+## Test
+It is extremely important that a plugin has many test to verify that is working properly.<br>
+A plugin without test will not be accepted, since it not inspire trust and the guarantee that the code will continue to work among different versions of its dependencies.
+
+We do not enforce any testing library, we use [`tap`](http://www.node-tap.org/) since it offers out of the box parallel testing and code coverage, but it is up to you choose your library of preference.
+
+## Code Linter
+It is not mandatory but we highly recommend to use a code linter in your plugin, it will ensure a consistent code style and help you avoid many errors.
+
+We do not enforce any linter, we use [`standard`](https://standardjs.com/) since it works without the need to configure it and is very easy integrate in your test suite.
+
+## Continuous Integration
+It is not mandatory, but if you release your code as open source [travis](https://travis-ci.org/) is free. A plugin with CI enabled will increase the trust with the plugin and help you with future changes or pull requests.<br>
+In addition you can enable services like [greenkeeper](https://greenkeeper.io/), that will help you maintain your dependencies up to date and discover if a new release of Fastify has some issues with your plugin.
```

Comment:
"[greenkeeper], that will help you mantain" => "[Greenkeeper] that will help you keep"